public/js/icinga/behavior/dashboards.js

```diff
@@ -152,33 +141,29 @@
                         case _this.widgetTypes.DashboardHome:
                             groupName = _this.widgetTypes.DashboardHome;
                             draggable = '.home-list-control';
-                            handle = '.home-list-control > h1';
+                            handle = 'h1 > .widget-drag-initiator';
                             break;
                         case _this.widgetTypes.Dashboard:
                             groupName = _this.widgetTypes.Dashboard;
                             draggable = '.dashboard-list-control';
-                            handle = '.dashboard-list-control > h1'
+                            handle = 'h1 > .widget-drag-initiator'
                             break;
                         case _this.widgetTypes.Dashlet:
                             groupName = _this.widgetTypes.Dashlet;
-                            if (sortable.matches('.content > .dashboard')) {
-                                draggable = '> .container';
-                            } else {
-                                draggable = '.dashlet-list-item';
-                            }
-
-                            handle = draggable;
+                            draggable = '.dashlet-list-item';
+                            handle = 'h1 > .widget-drag-initiator';
                     }
 
                     let options = {
-                        scroll     : true,
-                        invertSwap : true,
-                        delay      : 100,
-                        dataIdAttr : 'id',
-                        direction  : 'vertical',
-                        draggable  : draggable,
-                        handle     : handle,
-                        group      : { name : groupName }
+                        scroll      : true,
+                        invertSwap  : true,
+                        delay       : 100,
```

Comment:
The plan was to get rid of this delay then ;)

Reply:
🤦‍♂️